css/style.css

```diff
@@ -1186,3 +1192,7 @@ body button {
     width: 90%;
   }
 }
+.work-card_list {
```

Comment:
1) Use double underscore `__`
2) This is not list, this is list item

Final name: `work-card__list-item`